source/partials/release_notes/_release-17-2-0.md.erb

```diff
@@ -0,0 +1,119 @@
+#### Authorization Plugins (beta)
+
+Currently GoCD provides multiple ways of Authentication like File based Authentication, LDAP/AD Authentication and Plugin Based Authentication. Apart from Authentication, GoCD provides ability to restrict certain operations to specific Users and Roles and manage them. Currently, roles can be managed only through GoCD and it does not provide an ability to use ACL’s defined in systems used for authentication. With this limitation, administrators need to configure ACL's in multiples places.
+
+With the introduction of Authorisation Plugin(Beta)  GoCD is moving in a direction where Authentication/Authorisation is moved out of GoCD into plugins. With this, the Authorisation Plugins would be responsible to provide Identity Service. The plugins have the flexibility to use any identity service providers like LDAP, Github etc.
+
+
+[LDAP Authentication plugin](https://github.com/gocd/gocd-ldap-authentication-plugin) is in place built on this Authorization endpoint. Please checkout this feature and provide us feedback to enhance this endpoint.
+
+#### Feature Enhancements
```

Comment:
can we add few more things in feature enhancements ?(if there are any or else its fine)